### ISSUE TITLE
XLNet load error

### ISSUE THREAD

**Issue by Qbop981001** [PaddlePaddle/PaddleNLP#352]
When I import XLNet pretrained model with the following code:
```
from paddlenlp.transformers.xlnet.modeling import XLNetForTokenClassification
from paddlenlp.transformers.xlnet.tokenizer import XLNetTokenizer
do_train():
    .....
    model = XLNetForTokenClassification.from_pretrained("chinese-xlnet-base", num_classes=num_classes)
    tokenizer = XLNetTokenizer.from_pretrained("chinese-xlnet-base")
```
I meet this error:
![微信图片_20210510142615](https://user-images.githubusercontent.com/41386411/117615020-bd5d7d00-b19b-11eb-85fc-658ba9ba6fea.png)
However, when I replace XLNet with BertForTokenClassification, the error disappears.
So is this the problem of the XLNet framework?

**Comment by guoshengCS** [comment]
It is because the output of `XLNetForTokenClassification.forward` is a tuple, maybe you can try using `model()[0]` as the input of criterion. Sorry for the confusion of this.

**Comment by ZeyuChen** [comment]
@guoshengCS We need to unify the output of all the pre-trained models in PaddleNLP.

**Comment by yingyibiao** [comment]
Hi, we have recently unified the output format of XLNet Model with other models (like Bert). You can now replace with XLNet and get the same output format.

**Comment by yingyibiao** [comment]
#408

**Comment by ZeyuChen** [comment]
Issues solved by #408
